### PR TITLE
Mood page 1

### PR DESCRIPTION
**Pull request by NFranklin14**
mood page with cuisine buttons.

### CODE REVIEW

File: app/screens/landing/moodpage/MoodPage.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  },
+  list: {
+    alignItems: 'center',
+    padding: 0,
+    margin: 0,
+  },
+});
+
+function MoodPage({ navigation }) {
+  const mock =[{
```
Comment (mellava) [inline app/screens/landing/moodpage/MoodPage.jsx:31]: Update mocks/mockCuisines.js to reflect the 10 cuisines and import it here to keep mock data DRY.

File: app/App.jsx [hunk 2]
```diff
@@ -20,12 +21,13 @@ export default function App() {
         initialRouteName="Login"
```
Comment (fmirando) [inline app/App.jsx:21]: Resolved a merge conflict here. Remember to change initialRouteName back to login before sending a PR

File: app/screens/landing/moodpage/MoodButton.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    backgroundColor: '#61dafb',
+    color: '#20232a',
+    textAlign: 'center',
+    alignSelf: 'center',
+    fontSize: 30,
+    fontWeight: 'bold',
+  },
+});
+
+function MoodButton({ navigation, title, data }) {
```
Comment (fmirando) [inline app/screens/landing/moodpage/MoodButton.jsx:28]: Don't forget to add background images for each button

**Review by fmirando — APPROVED** [review]
Looks good Nathaniel! In App.jsx, don't forget to change initialRouteName back to login before submitting future PRs

File: app/screens/landing/moodpage/MoodPage.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 30,
+  },
+  touchable: {
+    width: 300,
+    borderWidth: 2,
+    borderColor: '#20232a',
+    borderRadius: 6,
+    backgroundColor: '#55d6bf',
```
Comment (mellava) [inline app/screens/landing/moodpage/MoodPage.jsx:16]: Use colors from constants file
```
import { COLORS } from '../../constants/colors';
...
COLORS.TURQUOISE

**Pull request merged**